esp/src/src-react/components/Details.tsx

```diff
@@ -213,6 +218,18 @@ export const Details: React.FunctionComponent<DetailsProps> = ({
                     />
                 });
                 break;
+            case "link":
+                field.href = field.href;
+                formFields.push({
+                    id: fieldID,
+                    label: field.label,
+                    field: <Link
+                        key={fieldID}
+                        href={field.href}
+                        target="_blank"
+                        style={{ paddingLeft: 8 }}>{field.href}</Link>
```

Comment:
@jeclrsg - is there theme / generic style I should be using this (paddingLeft)?

Reply:
I don't believe so. It looks like that is due to some default fluent [styling on inputs](https://github.com/microsoft/fluentui/blob/master/packages/react/src/components/TextField/TextField.styles.tsx#L304), which has the padding: "0 8px" in a generated class "field-###". This inline method seems reasonable to me. Another option could be to add a class on the table and set equivalent padding for anchors.

```
const contentStyles = mergeStyleSets({
    detailsTable: {
        "td a": {
            padding: "0 8px"
        }
    }
});
```

Reply:
Ok - will stick with inline for now...